src/adafruit_blinka/microcontroller/rockchip/rk3566/pin.py

```diff
@@ -169,7 +178,8 @@
 # ordered as i2cId, SCL, SDA
 i2cPorts = [
     (1, I2C1_SCL, I2C1_SDA),
-    (2, I2C2_SCL_M0, I2C2_SDA_M0),
+    # (2, I2C2_SCL_M0, I2C2_SDA_M0),
+    (2, I2C2_SCL_M1, I2C2_SDA_M1),
```

Comment:
I think this likely to cause issues for other devices that were using the I2C bus. I think we would want to selectively make this change only for devices that need it. 

You could add another branch to the if block down here: https://github.com/adafruit/Adafruit_Blinka/blob/1c4358371c4945d08fb208fa83e425f5b1af9d45/src/adafruit_blinka/microcontroller/rockchip/rk3566/pin.py#L199 to detect the relavent board(s) and then when found manipulate the `i2cPorts` list as needed for that device.